cmd/utils/flags.go

```diff
@@ -164,6 +164,18 @@ var (
 		Name:  "light",
 		Usage: "Enable light client mode",
 	}
+	IterativeOutputFlag = cli.BoolFlag{
+		Name:  "iterative",
+		Usage: "Print streaming json iteratively as json objects, delimited by lines",
+	}
+	ExcludeStorageFlag = cli.BoolFlag{
+		Name:  "nostorage",
+		Usage: "When dumping state, do not include storage (saves db lookups)",
```

Comment:
I don't think "When dumping state" is needed, since the flag is only used during dump.